addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -1303,6 +1306,16 @@ def check_same_picking_priority(self, pickings, mode="mobile"):
                 _logger.info(msg)
         return diff_priority_pickings
 
+    def get_log_batch_picking_flag(self):
+        """Get u_log_batch_picking configuration from warehouse and user name
+
+        Returns:
+            Boolean: u_log_batch_picking value
+        """
+        user = self.env["res.users"]
```

Comment:
Super nit: Capital User for an environment.

Reply:
done